jupyterlab_git/__init__.py

```diff
@@ -16,6 +34,8 @@ def _jupyter_server_extension_paths():
 def load_jupyter_server_extension(nbapp):
     """Load the Jupyter server extension.
     """
-    git = Git(nbapp.web_app.settings['contents_manager'])
+
+    user_custom_actions = JupyterLabGit(config=nbapp.config).actions
+    git = Git(nbapp.web_app.settings['contents_manager'], user_custom_actions)
```

Comment:
I would pass the all configuration object to anticipate other configuration settings.

```suggestion
    config = JupyterLabGit(config=nbapp.config)
    git = Git(nbapp.web_app.settings['contents_manager'], config)
```

Reply:
Addressed. Not I had to name it `_config` in `git.py` or a test was failing.